cns/NetworkContainerContract.go

```diff
@@ -78,10 +78,18 @@ type NICType string
 // NIC Types
 const (
 	InfraNIC NICType = "InfraNIC"
-	// Delegated VM NICs are projected from VM to container network namespace
-	DelegatedVMNIC NICType = "DelegatedVMNIC"
-	// BackendNIC NICs are used for infiniband nics on a VM
+	// DelegatedVMNIC are projected from VM to container network namespace
+	DelegatedVMNIC NICType = "FrontendNIC"
```

Comment:
to-check: will the name(string value) change break anything? thinking from DNC & DNC-RC point of view

Reply:
design has changed, this should not depend on the direct value sent by dnc now.